docs/release-notes/RELEASE-NOTES-0.9.0.md

```diff
@@ -6,45 +6,51 @@ It:
-In case you have installed either of them in a non-default path, you could use symlink(s) to work around the issue.
+The user configuration is documented in the [wiki](https://github.com/aeternity/epoch/wiki/User-provided-configuration).
+For specifying configuration using the Docker image, please refer to [its documentation](https://github.com/aeternity/epoch/blob/v0.9.0/docs/docker.md).
 
-To run on Ubuntu 16.04 you need to have a libsodium shared library (v1.0.16) in `/usr/local/lib/libsodium.so.23`. (`wget https://download.libsodium.org/libsodium/releases/libsodium-1.0.16.tar.gz`,
-unpack, then do `./configure && make && sudo make install && sudo ldconfig`)
+The node API - i.e. peer-to-peer network API and user API - is documented:
+* HTTP API endpoints are specified [online in swagger.yaml](https://github.com/aeternity/epoch/blob/v0.9.0/config/swagger.yaml);
+  * A JSON version of same specification is located in the node at path `lib/aehttp-0.1.0/priv/swagger.json`;
+  * An interactive visualization of the same specification is available [online](https://aeternity.github.io/epoch-api-docs/?config=https://raw.githubusercontent.com/aeternity/epoch/v0.9.0/apps/aehttp/priv/swagger.json).
+* WebSocket API endpoints are [specified online](https://github.com/aeternity/protocol/blob/epoch-v0.9.0/epoch/api/README.md);
+* The intended usage of the user API (HTTP and WebSocket) is [documented online](https://github.com/aeternity/protocol/blob/epoch-v0.9.0/epoch/api/README.md).
 
-Alternatively to the release binaries, you can use the published `aetrnty/epoch` Docker image by consulting its [documentation](https://github.com/aeternity/epoch/blob/v0.9.0/docs/docker.md).
+## Install node
 
-The user configuration is documented in the [wiki](https://github.com/aeternity/epoch/wiki/User-provided-configuration) though the instructions below contain easy-to-use examples.
+The instructions for installing a node using a release binary are in [the dedicated separate document](../../docs/installation.md).
 
-HTTP API endpoints are specified in the [swagger.yaml](https://github.com/aeternity/epoch/blob/v0.9.0/config/swagger.yaml); a swagger.json version of the same specification is present in the release binary at path `lib/aehttp-0.1.0/priv/swagger.json`, and its interactive visualization is available [online](https://aeternity.github.io/epoch-api-docs/?config=https://raw.githubusercontent.com/aeternity/epoch/v0.9.0/apps/aehttp/priv/swagger.json).
-WebSocket API endpoints are [specified](https://github.com/aeternity/protocol/blob/epoch-v0.9.0/epoch/api/README.md).
-
-The intended usage of the API (HTTP and WebSocket) of the node is [documented](https://github.com/aeternity/protocol/blob/epoch-v0.9.0/epoch/api/README.md).
+For installation of a node using the Docker image, please refer to [its documentation online](https://github.com/aeternity/epoch/blob/v0.9.0/docs/docker.md).
 
 ## Join the testnet
```

Comment:
I think this section ca be safely moved to configuration.md section

Reply:
You mean mainly the IP addresses of testnet?